### PR TITLE
feat: beautify things

### PR DESCRIPTION
**Pull request by SuperMarcomen**

### CODE REVIEW

File: web/src/lib/elements/ui/dataTable/data-table.svelte [hunk 1]
```diff
@@ -44,7 +44,7 @@
   export let additionalId: string = '';
   export let sortable = true;
   export let addButton = true;
-  export let pageSize = 15;
+  export let pageSize = 2;
```
Comment (danieldietzler) [inline web/src/lib/elements/ui/dataTable/data-table.svelte:47]: I don't think a default size of 2 is feasible

Comment (SuperMarcomen) [reply web/src/lib/elements/ui/dataTable/data-table.svelte]: You're right. I used that for testing but forgot removing it.

File: web/src/lib/elements/ui/dataTable/data-table.svelte [hunk 2]
```diff
@@ -266,7 +268,7 @@
             >
               {#each row.cells as cell (cell.id)}
                 <Subscribe attrs={cell.attrs()} let:attrs>
-                  <Table.Cell {...attrs}>
+                  <Table.Cell class="mammt" {...attrs}>
```
Comment (danieldietzler) [inline web/src/lib/elements/ui/dataTable/data-table.svelte:271]: WTH is "mammt"?

**Review by danieldietzler — APPROVED** [review]

File: web/src/lib/elements/ui/dataTable/data-table.svelte (hunk 2 above)
Comment (SuperMarcomen) [reply web/src/lib/elements/ui/dataTable/data-table.svelte:269]: I added a random class to see to which HTML element that cell would correspond and I forgot to remove it lol

**Pull request merged**